pyk/regression-new/issue-1633/2.test

```diff
@@ -1 +1 @@
-foo0\uffff\Uffffffff
+foo0\\uffff\\Uffffffff
```

Comment:
I don't entirely understand why this test was changed. The test was written to imitate the lexical syntax of C, and while it doesn't necessarily break anything to change it, if I don't understand why it was changed, it might be hiding a difference in behavior that might break real semantics.

Reply:
The issue here was that it was being interpreted as a Unicode character, but even though it can be parsed as Unicode, it isn't a valid Unicode. After discussing with Scott and Tamas, we'll go with a more limited modification that doesn't require a change to this test but prohibits any Unicode non-latin-1 encoded.

Reply:
Well it's good that we're not changing this test, so I guess it's moot, but I find your answer confusing because this test never used any unicode characters; that was a literal backslash character. You changed it from a literal backslash character to two literal backslash characters, which was strange enough that I felt the need to comment on it.